immich-app/immich
# show notifications on background backup errors

- shows a notification if the background backup failed (e.g. connecting to server failed) 
- shows only 1 notification regardless how often this failure occurred (automatic retries are performed in exponentially increasing intervals)
- shows individual error notifications if specific files could not be uploaded (because the API said so)
- automatically clears all errors once the background backup completed successfully

How to test this:

1. Have background backup enabled & configure immediate notifications in app settings
2. Shutdown the server (`docker compose stop immich-proxy` is enough)
3. Take a new picture
4. Wait until background backup runs; it should produce a notification
5. Restart server
6. Wait until the service retries the upload (60 seconds)
7. Check that error notification is gone and pictures uploaded to the server
